tests/clvm/test_program.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
```

Comment:
Do we want to use unittest here, not pytest?

Separately, if I understand correctly, a new test file will require a rerun of https://github.com/Chia-Network/chia-blockchain/blob/597068f400c0680a55a8595cf1ff1fb666686789/tests/build-workflows.py to actually be included in the CI (pending the reported inconsistency between the generated results and the template).

https://github.com/Chia-Network/chia-blockchain/blob/d59621c7469e269543c9942c575b8c6c94216592/.github/workflows/build-test-ubuntu-clvm.yml#L98

Reply:
@aqk Is there any reason the workflows list out every single file to be tested rather than just listing the enclosing directory? `py.test` has logic to find files that start with `test_*`, so simply using the directory could require the workflows to be changed less often.

Reply:
> Do we want to use unittest here, not pytest?

It's true that `py.test` has more features than `unittest`, but nothing fancy is used in these simple tests, and `py.test` run regular unit tests by default, so I think we're okay here.

Reply:
What can I say, I like consistency.  `:]`

Reply:
Hey, good question - yes, we can move these to just the directory.

The original reason I listed out all test files was that I thought that running:

`pytest dir`

Would run everything in `dir` as well as `dir/dir1` and `dir/dir2`

Reply:
It will recurse, from my quick look.  There's [`norecursedirs`](https://docs.pytest.org/en/6.2.x/reference.html#confval-norecursedirs), but that's a config file thing which would affect normal pytest usage.  There's also [`--ignore-glob`](https://docs.pytest.org/en/6.2.x/changelog.html#id597).  Would be nice if there was a `--no-recurse` option.

But, it looks like `pytest dir/test_*.py` works.